ci/docker/linux-apt-docs.dockerfile

```diff
@@ -76,6 +76,9 @@ RUN apt-get purge -y npm && \
     rm -rf /var/lib/apt/lists/* && \
     npm install -g yarn
 
+# Expose ARROW_PYTHON_VENV from BASE
+ENV ARROW_PYTHON_VENV /arrow-dev
```

Comment:
I think that we don't need this. (It should be inherited from the base image automatically.)

Reply:
Or how about defining `RETICULATE_PYTHON_ENV` here?

```dockerfile
ENV RETICULATE_PYTHON_ENV ${ARROW_PYTHON_VENV}
```